src/frontend/packages/tdbp/src/pages/Instructor/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Flexgrid, useLTIContext } from "@openfun/warren-core";
+import { Filters } from "../../components/Filters";
+import { Window } from "../../components/Window";
+import { ActiveActions } from "../../components/Activities";
+import { Radar } from "../../components/Radar";
+import { StudentsComparison } from "../../components/StudentsComparison";
+import { TdbpFiltersProvider } from "../../contexts";
+
+/**
+ * A React component responsible for rendering a dashboard overview of student activity
+ * statistics.
+ *
+ * @returns {JSX.Element} The JSX for the students activity overview dashboard.
+ */
+export default () => {
+  const { appData } = useLTIContext();
+
+  const tokenPayload = appData.access.split(".");
+  const decodedPayload = JSON.parse(atob(tokenPayload[1]));
```

Comment:
Or you can use this lib : https://www.npmjs.com/package/jwt-decode
And instead of doing this, you can also decode it in a context and make it available everywhere in the application. Decoding a token is time consuming, doing it once is better IMO